src/pages/index.js

```diff
@@ -20,100 +24,178 @@ import {
 /*---------------------------------------------------*/
 
 /*-- Add Modal --*/
-const addModal = document.querySelector("#add-modal");
 const addModalForm = document.querySelector("#add-modal-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms